lte/protos/subscriberdb.proto

```diff
@@ -242,7 +242,7 @@ message SyncSubscribersResponse {
   repeated SubscriberDigestWithID per_sub_digests = 3;
   // to_renew is the set of all subscriber data modified/added on the server side,
   // indexed by their IDs (IMSI).
```

Comment:
update the description to reflect the change as this is not indexed by IMSI anymore

Reply:
Thanks for the catch!